orders/tests.py

```diff
@@ -0,0 +1,70 @@
+import datetime
+from django.utils import timezone
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+from .models import Order
+from .models import StatusOptions
+import pytest
+
+STARTING_DATE = timezone.now()
+FINISHING_DATE = timezone.now() + datetime.timedelta(days=3)
+PRICE_PER_DAY = 100
+STATUS = StatusOptions.Pending
```

Comment:
None of those constants is used more than once, you can just use them in the dedicated place instead of declaring them here (for now I don't see a benefit for it)

Reply:
> None of those constants is used more than once, you can just use them in the dedicated place instead of declaring them here (for now I don't see a benefit for it)

Changed it.